main_test.go

```diff
@@ -328,7 +340,7 @@ func createRootfs(dir string) error {
 
 // makeOCIBundle will create an OCI bundle (including the "config.json"
```

Comment:
does that func name in the comment now need fixing?

Reply:
I see your point, but I think it's still fair to say it's making the bundle; it's just "making" it from a copy now ;)

Reply:
I mean the func name itself - this func is no longer called `makeOCIBundle` - now it is `realMakeOCIBundle`, no? :-)